x-aeson/ambiata-x-aeson.cabal

```diff
@@ -15,7 +15,11 @@ library
                        aeson                           >= 0.8        && < 0.12
                      , base                            >= 3          && < 6
                      , bytestring                      == 0.10.*
+                     , containers                      == 0.5.*
                      , text                            >= 1.1        && < 1.3
+                     , time                            >= 1.5        && < 2
```

Comment:
Can't really drop `thyme` in, any other library you use that takes `UTCTime` (`tzdata` comes to mind) can't be used without a lot of duct tape